lib/uksched/sched.c

```diff
@@ -392,3 +392,26 @@ void uk_sched_dumpk_threads(int klvl, struct uk_sched *s)
 			  (t->flags & UK_THREADF_UKTLS)    ? 'T' : '-');
 	}
 }
+
+UK_SYSCALL_R_DEFINE(int, sched_getaffinity, int, pid, long, cpusetsize, unsigned long*, mask)
```

Comment:
Wrap this to ~80 characters.
```suggestion
UK_SYSCALL_R_DEFINE(int, sched_getaffinity, int, pid, long, cpusetsize,
                                           unsigned long*, mask)
```